code/index.html

```diff
@@ -1,30 +1,167 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <title>Bookbinding Courses</title>
     <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
+    <!--Google Fonts-->
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;1,400&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+     <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <div class="hero">
+        <video width="" height="" playsinline autoplay muted loop poster="" class="bgvideo">
+          <source src="pageturner.mp4" type="video/mp4">
+        </video>
+      </div>
+      
+      <div class="logo">
+        <img class="imglogo" src="BC_logoV2.png" alt="logotype for Bookbinders Club" />
+      </div>
 
-    <!-- Signup form -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      
+      <nav class="nav">
+        <a class="nav-item" href="#" >About</a>
+        <a class="nav-item" href="#">Courses</a>
+        <a class="nav-item" href="#">Merchandise</a>
+        <a class="nav-item" href="#">Contact</a>
+      </nav>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="headline">
+        <h1>A Forgotten Craftsmanship</h1>
+        <p>Learn Bookbinding Today!</p>
+      </section>
+    </header>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <div class="main">
+
+      <!-- Signup form -->
+      <section class="form">
+        <form action="http://httpbin.org/anything" method="POST">
+        <h2>Sign up here!</h2>
+        <h5>To get more information and notification alernatives.</h5>
+     
+        <div class= "text-input">
+          <label class="name">
+            Name<br>
+            <input class="textfield" type="text" name="message" required></input><br>
```

Comment:
Inputs are self-closing tags because they don't have any nested content, so you can write like this. And maybe it makes more sense if the `name` attribute was "name"? :)
```
<input
  required
  class="textfield"
  type="text"
  name="name"
/>
```